### PR TITLE
Sldss 26214 add people api to api docs to get user info by uids

### PR DESCRIPTION
**Pull request by achebanets**

### CODE REVIEW

File: spec/openapi.yaml [hunk 1]
```diff
@@ -97,6 +97,10 @@ tags:
       Locales are the language and country pairs that are used to identify the
       source content and the desired target locale for your translated content.
 
+  - name: Peoples
```
Comment (AlexArcher) [inline spec/openapi.yaml]: People

File: spec/openapi.yaml [hunk 2]
```diff
@@ -97,6 +97,10 @@ tags:
...
+    description: |-
+      People - it is user of the system who works, processes, translates context.
```
Comment (AlexArcher) [inline spec/openapi.yaml]: All users in Smartling platform

File: spec/openapi.yaml [hunk 3]
```diff
@@ -775,6 +779,67 @@ paths:
         500:
           $ref: '#/components/responses/Error500ResponseDefinition'
 
+
+#
+# Peoples API
```
Comment (AlexArcher) [inline spec/openapi.yaml]: People API

File: spec/openapi.yaml (hunk 2 above)
Comment (bbristow-smartling) [inline spec/openapi.yaml]: All Smartling users, who process and translate content with the Smartling platform.

File: spec/openapi.yaml (hunk 3 above)
Comment (bbristow-smartling) [reply spec/openapi.yaml]: Agreed - People API.

File: spec/openapi.yaml [hunk 4]
```diff
@@ -775,6 +779,67 @@ paths:
...
+                    List of user uids.
+                  example:
+                    - "ac48ht4qy"
+					
+                  items:
+                    type: string
+                  type: array
+                excludeProfileImage:
+                  description: >-
+                    Exclude user profile image. It is optimization to avoid fetching user's profile image
```
Comment (bbristow-smartling) [inline spec/openapi.yaml]: As a performance optimization, exclude the user's profile image.

File: spec/openapi.yaml (hunk 2 above)
Comment (bbristow-smartling) [reply spec/openapi.yaml]: Smartling users, who process and translate content with the Smartling platform.

File: spec/openapi.yaml (hunk 1 above)
Comment (achebanets) [reply spec/openapi.yaml]: Done

File: spec/openapi.yaml (hunk 2 above)
Comment (achebanets) [reply spec/openapi.yaml]: Done

File: spec/openapi.yaml (hunk 3 above)
Comment (achebanets) [reply spec/openapi.yaml]: Done

File: spec/openapi.yaml (hunk 4 above)
Comment (achebanets) [reply spec/openapi.yaml]: Done

File: spec/openapi.yaml [hunk 5]
```diff
@@ -775,6 +779,67 @@ paths:
...
+# Peoples API
+#
+
+  /people-api/v2/users:
+    post:
+      summary: Get list of user information
+      tags:
+      - Peoples
+      description: >
+        People is user of the system who process, translate context
```
Comment (bbristow-smartling) [inline spec/openapi.yaml]: Smartling users, who process and translate content with the Smartling platform.

File: spec/openapi.yaml [hunk 6]
```diff
@@ -775,6 +779,67 @@ paths:
...
+
+#
+# Peoples API
+#
+
+  /people-api/v2/users:
+    post:
+      summary: Get list of user information
+      tags:
+      - Peoples
```
Comment (bbristow-smartling) [inline spec/openapi.yaml]: People

File: spec/openapi.yaml [hunk 7]
```diff
@@ -14892,6 +14957,50 @@ components:
...
+                      type: integer
+                  required:
+                    - totalCount
+                    - items
+                  type: object
+              type: object
+
   responses:
     Error400ResponseDefinition:
       description: Provided request parameters are invalid.
```
Comment (bbristow-smartling) [inline spec/openapi.yaml]: The request parameters are invalid.

File: spec/openapi.yaml [hunk 8]
```diff
@@ -775,6 +779,67 @@ paths:
...
           $ref: '#/components/responses/Error500ResponseDefinition'
 
+
+#
+# Peoples API
+#
+
+  /people-api/v2/users:
+    post:
+      summary: Get list of user information
```
Comment (bbristow-smartling) [inline spec/openapi.yaml]: Get a list of user information

File: spec/openapi.yaml (hunk 5 above)
Comment (achebanets) [reply spec/openapi.yaml]: It was already done.

**Pull request merged**